packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -5377,7 +5377,7 @@ const tests = {
             `The 'handleNext' function makes the dependencies of ` +
             `useEffect Hook (at line 11) change on every render. ` +
             `Move it inside the useEffect callback. Alternatively, ` +
-            `wrap the 'handleNext' definition into its own useCallback() Hook.`,
+            `wrap the construction of 'handleNext' in its own useCallback() Hook.`,
```

Comment:
I don't think "construction" is a common term in this case. People don't think about declaring functions as "constructing" them.

Reply:
Yeah, I wasn't 100% happy with this either. Can you think of a term which would encompass both declaration and any subsequent mutation needed to create an object/function? If the term needs to be different for objects/arrays than for functions that's fine.

Reply:
- wrap the [name] function body
- wrap the [name] object creation

?

Reply:
This still needs to be changed.

Reply:
I still think we should keep "definition" for functions. And maybe "assignment" for everything else.

Reply:
I worry that suggesting that they "wrap the assignment" implies that its the right-hand-side of the assignment expression that needs to be wrapped, and does not convey that any subsequent mutations of that object _also_ need to get wrapped in the same `useMemo`.

Maybe it's the best we can do, so I'll go with that for now, but I wish there were a way to clearly communicate that given:

```
const foo = []; // <== Assignment
foo.push(bar);
```

We want them to wrap _both_ the assignment _and_ the mutation (like this):

```
// GOOD
const foo = useMemo(() => {
   const f = [];
   f.push(bar);
}, [bar]);
```

And not _just the assignment_ (like this): 

```
// BAD
const foo = useMemo(() => [], []);
foo.push(bar);
```

Reply:
"creation" then?

Reply:
"initialization" is also fine by me.

Reply:
Ah, I like initialization. I'll go with that.

Reply:
Done in https://github.com/facebook/react/pull/19590/commits/2b7ae4b79faed0d182b0aaa468b79b63171dc437